### PR TITLE
distutils-r1.eclass: Split distutils_wheel_install + min dep version bump

### PR DESCRIPTION
**Pull request by mgorny**
CC @gentoo/python

### CODE REVIEW

**Comment by mgorny** [comment]
(though it'll probably make sense to wait for 3.11.0b4 and update the dep before pushing, as it brings sqlite fix needed for peewee)

**Review by arthurzam — APPROVED** [review]
@mgorny Looks very good, thank you!

**Comment by arthurzam** [comment]
> No edo?

Actually I thought the same thing, so I checked the implementation of `edo`, and it uses `einfo` instead of `printf` here. From the way the log is outputted, how info is shown, etc - I think that `printf` is better here.

**Comment by mgorny** [comment]
> No edo?

Nah, let's stay in Heisei.

File: dev-python/sphinx-basic-ng/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>python@gentoo.org</email>
+	</maintainer>
+	<upstream>
```
Comment (thesamesam) [inline dev-python/sphinx-basic-ng/metadata.xml:7]: ALLARCHES?

Comment (mgorny) [reply dev-python/sphinx-basic-ng/metadata.xml:7]: I guess makes sense.

**Comment by thesamesam** [comment]
Merged.

**Pull request closed without merging**